ui/config/content-security-policy.js

```diff
@@ -0,0 +1,18 @@
+module.exports = function (environment) {
+  return {
+    delivery: ['header', 'meta'],
+    enabled: environment !== 'production',
+    failTests: true,
+    policy: {
+      'default-src': ["'none'"],
+      'script-src': ["'self'"],
+      'font-src': ["'self'"],
+      'connect-src': ["'self'"],
+      'img-src': ["'self'", 'data:'],
+      'style-src': ["'unsafe-inline'", "'self'"],
+      'media-src': ["'self'"],
+      'form-action': ["'none'"],
+    },
+    reportOnly: false,
+  };
+};
```

Comment:
I followed [the deprecation guide](https://github.com/rwjblue/ember-cli-content-security-policy/blob/master/DEPRECATIONS.md) after upgrading `ember-cli-content-security-policy` to latest.

Reply:
Nice!